.github/workflows/test.yml

```diff
@@ -16,7 +16,7 @@ defaults:
 jobs:
   e2e-test:
     if: github.repository_owner == 'getsentry'
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
bumped this while at it

Reply:
nit: Imo we should split this into a separate PR, since I could imagine cases where we want to only roll this bump back.